package.json

```
@@ -13,6 +13,7 @@
   "scripts": {
     "build": "lerna run build --parallel",
     "lint": "lerna run lint --stream --no-bail",
```

Should this all-encompasing lint command be updated to? 

```suggestion
    "lint": "npm run lint:markdown && lerna run lint --stream --no-bail",
```